app/services/encryption/password_verifier.rb

```diff
@@ -40,12 +40,18 @@ def digest(password:, user_uuid:)
       encrypted_password = encrypt_password(
         password: password, user_uuid: user_uuid, salt: salt, cost: cost,
       )
-      PasswordDigest.new(
+      single_region_digest = PasswordDigest.new(
         encrypted_password: encrypted_password, password_salt: salt, password_cost: cost,
       ).to_s
+
+      RegionalCiphertextPair.new(
+        single_region_ciphertext: single_region_digest,
+        multi_region_ciphertext: nil,
+      )
```

Comment:
Since this method now returns a digest_pair, would it make sense to rename the method to `digest_pair`? I only saw it being used from user_access_key_overrides.rb above - other uses of `digest` in this file look like they're local variables hiding the method.

Reply:
That is an interesting one. I am happy to rename it but it is a little uncomfortable. For some reason in my brain "digest" is a verb here and a noun in all the other places. Maybe something like "create_digest_pair"? I'll think on it and go with "digest_pair" if I can't come up with anything better since it does in face return a digest pair.